discovery-client/src/main/java/io/micronaut/discovery/client/DiscoveryClientCacheConfiguration.java

```diff
@@ -35,9 +35,9 @@
  * @since 1.0
  */
 @Named(CACHE_NAME)
-@ConfigurationProperties(CACHE_NAME)
+@ConfigurationProperties(CacheConfiguration.PREFIX + "." + CACHE_NAME)
 @Requires(property = DiscoveryClientCacheConfiguration.SETTING_ENABLED, notEquals = StringUtils.FALSE)
-public class DiscoveryClientCacheConfiguration extends DefaultCacheConfiguration implements Toggleable {
+public class DiscoveryClientCacheConfiguration extends CacheConfiguration implements Toggleable {
```

Comment:
This could be considered a breaking change, however it doesn't make sense to extend `@EachProperty` with `@ConfigurationProperties`. It would not be possible to configure the same key differently. For example:

`team.cubs.name: Cubs, team.cubs.manager.name: Jeff`